lib/players/human.rb

```diff
@@ -0,0 +1,11 @@
+require 'pry'
+
+module Players
+    class Human < Player
+        def move(board)
+            puts "Enter your move: (1-9):"
+            player_move = gets.strip
+            return player_move
+        end
+    end
+end
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Enter your move: (1-9):"
  gets.strip
end
```